buf.yaml

```diff
@@ -10,6 +10,7 @@ lint:
     - UNARY_RPC
     - COMMENT_FIELD
     - PACKAGE_DIRECTORY_MATCH
+    - ENUM_ZERO_VALUE_SUFFIX
```

Comment:
```suggestion
  enum_zero_value_suffix: _DEFAULT
```

Reply:
@alexanderbez what do you think of using the `_DEFAULT` suffix rather than just disabling `ENUM_ZERO_VALUE_SUFFIX`? We would then need to change all our existing enum's to end in `_DEFAULT` though